misoc/integration/soc_core.py

```diff
@@ -88,6 +88,8 @@ def __init__(self, platform, clk_freq,
         assert(self.cpu_dw, cpu_bus_width)
         self.config["DATA_WIDTH_BYTES"] = self.cpu_dw//8
 
+        self.endianness = getattr(self.cpu, "endianness", "big")
```

Comment:
I wouldn't use a default - just define the endianness in each CPU.

Reply:
Also this can just be ``self.cpu.endianness`` elsewhere, no need to copy the attribute?

Reply:
0f79c39